internal/util/cephcmds.go

```diff
@@ -219,27 +219,27 @@ func RemoveOMapKey(ctx context.Context, monitors string, cr *Credentials, poolNa
 // CreateObject creates the object name passed in and returns ErrObjectExists if the provided object
 // is already present in rados
 func CreateObject(ctx context.Context, monitors string, cr *Credentials, poolName, namespace, objectName string) error {
-	// Command: "rados <options> create objectName"
-	args := []string{
-		"-m", monitors,
-		"--id", cr.ID,
-		"--keyfile=" + cr.KeyFile,
-		"-c", CephConfigPath,
-		"-p", poolName,
-		"create", objectName,
+	conn, err := connPool.Get(monitors, cr.ID, cr.KeyFile)
```

Comment:
how about making use of funcitons already defined in https://github.com/ceph/ceph-csi/blob/master/internal/util/connection.go file?